canonical-web-and-design/microk8s.io
# Search for docs

## Done

- Add search for docs
- Update to dart sass

## QA
- Go to https://microk8s-io-384.demos.haus/docs
- Search using the new search box, try a term that yields results e.g. `application`
- You may have next button for pagination, check that it works

## Issue / Card

Fixes #383
